src/app.service.ts

```diff
@@ -1,12 +1,13 @@
 import { Injectable } from '@nestjs/common';
 import { LogMessageFormat } from "logging-format";
 import { AlertConverterService } from './alert-converter/alert-converter.service';
+import { ConfigService } from '@nestjs/config';
 const { Kafka } = require('kafkajs');
 
 //This services sends the LogMessages to the Queue and the issue-creator consumes them.
 const kafka = new Kafka({
   clientId: 'prometheus-alert-converter',
-  brokers: ['localhost:9092']
+  brokers: [new ConfigService().get<string>("KAFKA_URL", "localhost:9092")]
```

Comment:
hardcoded urls are evil. lässt sich das in eure `.env` packen?

Reply:
ist drinne bei mir, bei dir nicht?

Reply:
du hast natürlich recht. selbes missverständnis meinerseits, wie auch oben schon.